db/periodic_task_scheduler.cc

```diff
@@ -26,23 +26,28 @@ static const std::map<PeriodicTaskType, uint64_t> kDefaultPeriodSeconds = {
     {PeriodicTaskType::kPersistStats, kInvalidPeriodSec},
     {PeriodicTaskType::kFlushInfoLog, 10},
     {PeriodicTaskType::kRecordSeqnoTime, kInvalidPeriodSec},
+    {PeriodicTaskType::kTriggerCompaction, 12 * 60 * 60}  // 12 hours
```

Comment:
I thought about if there could be cases in which someone might need a more frequent interval, but this is really for applying compaction filter TTLs or ensuring privacy requirements. So this should be generally fine (even beyond a patch release)